pocket_lab/pl_gui_views.c

```diff
@@ -177,6 +193,15 @@ static adi_fft_data_to_volt_conv data_to_volt_wrt_vref;
  * binary data */
 static adi_fft_code_to_straight_bin_conv code_to_straight_binary;
 
+/* Pocket Lab Instance */
+static struct pl_gui_init_param pl_instance =  {
+	.event = NULL,
+	.event1 = NULL
+};
+
+/* Local copy of init params */
```

Comment:
should this be "global" copy?

Reply:
Local here means local copy needed for the file ie local copy of the init params passed from the application layer. HHave used the same comments for other such copies as well.  Do you suggest a better comment can be added?